### ISSUE TITLE
Build into dist root

### ISSUE THREAD

**Issue by MetaMmodern** [JohnPremKumar/vite-plugin-favicons-inject#14]
Hi, I want to put the result of your plugin under root of dist and not into dist/assets. How do I do that? How do I postpocess the result of your plugin?

I need at least favicon.ico to be accessible through my.website.com/favicon.ico, and not my.website.com/assets/favicon.ico.

**Comment by JohnPremKumar** [comment]
This defaults to vite assest directory option. You can control it using the vite config as mentioned [here](https://vitejs.dev/config/build-options.html#build-assetsdir)

**Comment by MetaMmodern** [comment]
@JohnPremKumar yes, but then all my assets will be in one folder, and I wanted favicon related stuff to be in root(dist/), while other assets in dist/assets.

~~Anyway, I'm not going to use this library, because favicons library doesn't generate faicons for Safari MacOS. And I need them. So I'll just use https://realfavicongenerator.net/~~

Update: actually this library is much better than provided website, I just encountered a caching issue. The favicons are generated correctly. But I still need some option to build into root, and not into assets. From description of Vite I see that it's hardly posible. But keeping assets in one folder is also something that is important. But according to that website my favicon.ico has to be in root. So idk, I'm on a crossroads now, what to do.

**Comment by JohnPremKumar** [comment]
I will check about the part to provide the option override the asset dir. In the meantime, you can try using the plugins like vite-plugin-static-copy to copy favicon file in to your desired location.

**Comment by MetaMmodern** [comment]
Actually I just used that website to create an ico and put it into public folder.It's "different" from what your plugin generates, but that's better than nothing. But thanks, I'll take a look at what you shared.
